experimental/dds/tree2/src/feature-libraries/default-schema/defaultChangeFamily.ts

```diff
@@ -81,11 +83,19 @@ export interface IDefaultEditBuilder {
 
 	/**
 	 * @param field - the sequence field which is being edited under the parent node
+	 *
+	 * @param shapeInfo - optional shape information used for schema based chunk encoding.
```

Comment:
This file has tests which should be updated to include testing this new option. The main point of exposing this is so that you can test schema compressed encoding more end to end, since no user of this API is actually going to provide this shapeInfo.

Reply:
I added 2 tests to the `defaultChangeFamily.spec.ts` file, where I repeated the existing insert tests, but passing in the shapeInfo to the sequenceField this time. The tests pass (as it has the expected jsonableTree), but should additional checks be made to make sure that the encoded format was using schema based encoding?